comps/cores/mega/http_service.py

```diff
@@ -96,6 +97,7 @@ async def _get_statistics():
     async def initialize_server(self):
         """Initialize and return HTTP server."""
         self.logger.info("Setting up HTTP server")
+        logging.getLogger("uvicorn.access").addFilter(lambda record: "/v1/health_check" not in record.getMessage())
```

Comment:
uvicorn config the logging settings at uvicon.Config [initialization time ](https://github.com/encode/uvicorn/blob/0.32.0/uvicorn/config.py#L357), can we try to move this after the uvicorn logging initialization, i.e. before line 136, to avoid potential conflict between the future uvicorn's default logging config?

Reply:
Fixed